fusion.ml

```diff
@@ -439,6 +442,23 @@ let rec type_subst i ty =
   let dest_eval = 
     function (Eval(e,ty)) -> e,ty | _ -> failwith "dest_eval: not an eval"
 
+(* ------------------------------------------------------------------------- *)
+(* Add to theorem & not-effective lists                                      *)
+(* ------------------------------------------------------------------------- *)
+
+  let addThm tm = match tm with 
+      | Sequent(asl,c) -> if (forall (fun a -> is_proven_thm a) asl) then 
```

Comment:
I much prefer to put the nested if on the next line, and indent it 4 from the `|` (which is 2 from the `Sequent`). That way the code isn't so wide, with lots of wasted space on the left.